From 24ad069ec2cb5bc95c9d965d44354424159cb92e Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 12:58:45 -0400
Subject: [ethernet] qlge: Convert /n to \n

Message-id: <1408625927-3701-15-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88537
O-Subject: [PATCH RHEL 7.1 14/16] qlge: Convert /n to \n
Bugzilla: 1089357
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Joe Perches <joe@perches.com>

Bug 1089357 (https://bugzilla.redhat.com/show_bug.cgi?id=1089357)

net commit id a42c3a28e8c2f071749a051f3afdbc1777418a07

Use a newline character appropriately.

Signed-off-by: Joe Perches <joe@perches.com>
Acked-by: Jitendra Kalsaria <jitendra.kalsaria@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlge/qlge_main.c b/drivers/net/ethernet/qlogic/qlge/qlge_main.c
index 91f161d..6aaf135 100644
--- a/drivers/net/ethernet/qlogic/qlge/qlge_main.c
+++ b/drivers/net/ethernet/qlogic/qlge/qlge_main.c
@@ -3603,7 +3603,7 @@ static int ql_request_irq(struct ql_adapter *qdev)
  }
  return status;
 err_irq:
- netif_err(qdev, ifup, qdev->ndev, "Failed to get the interrupts!!!/n");
+ netif_err(qdev, ifup, qdev->ndev, "Failed to get the interrupts!!!\n");
  ql_free_irq(qdev);
  return status;
 }
-- 
1.7.1